tests/jni_api.rs

```diff
@@ -413,6 +413,109 @@ pub fn get_byte_array_elements_auto() {
     assert_eq!(res[2], 4);
 }
 
+#[test]
+pub fn java_get_long_array_elements() {
+    let env = attach_current_thread();
+
+    // Create original Java array
+    let buf: &[i64] = &[1, 2, 3];
+    let java_array = env
+        .new_long_array(3)
+        .expect("JNIEnv#new_long_array must create a java array with given size");
+
+    // Insert array elements
+    let _ = env.set_long_array_region(java_array, 0, buf);
+
+    // Get array elements
+    let (ptr, _is_copy) = env.get_long_array_elements(java_array).unwrap();
+
+    // Check
+    assert_eq!(unsafe { *ptr.offset(0) }, 1);
+    assert_eq!(unsafe { *ptr.offset(1) }, 2);
+    assert_eq!(unsafe { *ptr.offset(2) }, 3);
+
+    // Modify
+    unsafe {
+        *ptr.offset(0) += 1;
+        *ptr.offset(1) += 1;
+        *ptr.offset(2) += 1;
+    }
+
+    // Release
+    env.release_long_array_elements(
+        java_array,
+        unsafe { ptr.as_mut().unwrap() },
+        ReleaseMode::CopyBack,
+    )
+    .expect("JNIEnv#release_long_array_elements must release Java array");
+
+    // Confirm modification of original Java array
+    let mut res: [i64; 3] = [0; 3];
+    env.get_long_array_region(java_array, 0, &mut res).unwrap();
+    assert_eq!(res[0], 2);
+    assert_eq!(res[1], 3);
+    assert_eq!(res[2], 4);
+}
+
+#[test]
+pub fn get_long_array_elements_auto() {
```

Comment:
Can we also test explicit `commit` (with an assumption that an array is a copy)?